src/frontend/komponenter/Task/TaskElement.tsx

```diff
@@ -9,8 +9,8 @@ interface IProps {
 const TaskElement: React.FC<IProps> = ({ innhold, label }) => {
     return (
         <div className={'taskelement'}>
-            <Element children={`${label}:`} />
-            <Normaltekst children={innhold} />
+            <Label size={'small'}>{label}:&nbsp;</Label>
```

Comment:
```suggestion
            <Label as="p" size={'small'}>{label}:&nbsp;</Label>
```
Jeg tror denne bør ha en `as="p"` ettersom den ellers blir en `<label>`, som vi bare skal bruke sammen med skjemaelementer